Rakefile

```diff
@@ -31,20 +31,20 @@ task :rubocop do
   Rake::Task['test:coding_guidelines'].invoke
 end
 
+require 'cucumber/rake/task'
+
 namespace :test do
-  desc 'Run cucumber tests'
-  task :cucumber do
-    sh 'bundle exec cucumber'
-  end
+  Cucumber::Rake::Task.new
 
-  desc 'Run cucumber tests which are "WORK IN PROGRESS" and are allowed to fail'
-  task :cucumber_wip do
-    sh 'bundle exec cucumber -p wip'
+  Cucumber::Rake::Task.new(:cucumber_wip, 'Run cucumber tests which are "WORK IN PROGRESS" and are allowed to fail') do |t|
```

Comment:
Right now this project's max line length is 169. (I do not know why).
But I think considering default line max length 80 is better. (inserting line break)
https://github.com/cucumber/aruba/blob/master/.rubocop.yml#L39